src/components/UserInfo/UserInfo.tsx

```diff
@@ -1 +1,22 @@
-export const UserInfo = () => {};
+import React from 'react';
+
+import './Userinfo.scss';
+
+import { User } from '../../types/User';
+
+type Props = {
+  user: User;
+};
+
+export const UserInfo: React.FC<Props> = ({ user }) => {
+  const {
+    name,
+    email,
+  } = user;
```

Comment:
Also, you could write it in one line in this case
```suggestion
  const { name, email } = user;
```